src/Scene/NodeProcess.js

```diff
@@ -99,6 +100,7 @@ NodeProcess.prototype.subdivideNode = function subdivideNode(node, camera, param
                 // update Imagery wmts
                 for (const layer of colorLayers) {
                     if (layer.tileInsideLimit(child, layer)) {
+                        TiledImageTools.computeTileWMTSCoordinates(child, layer);
```

Comment:
[WMTS *is* a protocol](https://en.wikipedia.org/wiki/Web_Map_Tile_Service). 

Yes, we support WSG84G TileMatrixSet + we kind of support PM TileMatrixSet. But actually, everything fits together because our geometry follow the same tile scheme. It is specific to the globe, and even more : to a IGN-homemade TileMatrixSet. So this still needs renaming, but I'm gonna start a new discussion in the right place (where this method is defined) proposing something in my next review.